### ISSUE TITLE
Will it be possible in the future to add a time filter?

### ISSUE THREAD

**Issue by vanBersie** [surveyjs/survey-library#594]
**Requested Feature**

On [https://dxsurvey.com/Home/MySurveys](https://dxsurvey.com/Home/MySurveys) an option that the survey is just available during a certain time span would be really cool, so that the creator can set a start date and an end date and when a user user comes to the site with the survey before it starts it won't be initialized and when he comes on that site after the survey is over there could be an option to set a text that says something like - "the survey is already over" and another option that the survey won't initialize at all or just display the results.

**Comment by andrewtelnov** [comment]
@vanBersie Yes, it makes sence. I have added this suggestion to our TODO list.

Thank you,
Andrew

**Comment by vanBersie** [comment]
@andrewtelnov Sound great! I'm looking forward to it. Thank you very much! :)
